tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,29 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+
+class BugzillaLite(Base):
+
+    name = 'Bzlite'
+    _given_username = ('css selector', "#login input[type='email']")
+    _given_password = ('css selector', "#login input[type='password']")
+    _button_login = ('css selector', "#login input[type='submit']")
+    # home frame
+    home_frame = marionette.find_element('css selector', 'div.homescreen iframe')
+    marionette.switch_to_frame(home_frame)
+    #launch
+    def launch (self):
```

Comment:
Now that the page class inherit from `Base`, we can use `Base.launch()` [like here](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py#L23).